source/dagger/gameplay/team_game/level_generator.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "core/core.h"
+
+namespace level_generator
+{
+    namespace smiljana
```

Comment:
a bit weird to split code like that, hope you will merge with more meaningful namespaces later. 
You can leave names in comments thou.

Reply:
Yes, we organized it like this so each of us could build their levels and test them and not worry too much about function names. These inner namespaces will be removed and we will settle on singular set of functions in the end.